cloud_AWS/terraform/module/examples/multiple-vpc-regions/bucket/variables.tf

```diff
@@ -0,0 +1,13 @@
+variable "vpc_id_list" {
+  type        = list(string)
+  default     = [""]
```

Comment:
can the list be empty and the example will still work?

Reply:
The error will occur. Ill changed it.

Reply:
The error will occur. Ill changed it.

Reply:
Variable removed so comment resolved